api_tester/lib/errors_api_test.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/services.dart';
+import 'package:purchases_flutter/errors.dart';
+
+// ignore_for_file: unused_element
+// ignore_for_file: unused_local_variable
+class _ErrorsApiTest {
+  void _checkPurchasesErrorCode(PurchasesErrorCode code) {
+    switch(code) {
+      case PurchasesErrorCode.unknownError:
+      case PurchasesErrorCode.purchaseCancelledError:
+      case PurchasesErrorCode.storeProblemError:
+      case PurchasesErrorCode.purchaseNotAllowedError:
+      case PurchasesErrorCode.purchaseInvalidError:
+      case PurchasesErrorCode.productNotAvailableForPurchaseError:
+      case PurchasesErrorCode.productAlreadyPurchasedError:
+      case PurchasesErrorCode.receiptAlreadyInUseError:
+      case PurchasesErrorCode.invalidReceiptError:
+      case PurchasesErrorCode.missingReceiptFileError:
+      case PurchasesErrorCode.networkError:
+      case PurchasesErrorCode.invalidCredentialsError:
+      case PurchasesErrorCode.unexpectedBackendResponseError:
+      case PurchasesErrorCode.receiptInUseByOtherSubscriberError:
+      case PurchasesErrorCode.invalidAppUserIdError:
+      case PurchasesErrorCode.operationAlreadyInProgressError:
+      case PurchasesErrorCode.unknownBackendError:
+      case PurchasesErrorCode.invalidAppleSubscriptionKeyError:
+      case PurchasesErrorCode.ineligibleError:
+      case PurchasesErrorCode.insufficientPermissionsError:
+      case PurchasesErrorCode.paymentPendingError:
+      case PurchasesErrorCode.invalidSubscriberAttributesError:
+      case PurchasesErrorCode.logOutWithAnonymousUserError:
+      case PurchasesErrorCode.configurationError:
+      case PurchasesErrorCode.unsupportedError:
```

Comment:
If there are missing `case` on these enum tests, it will be caught during the `flutter analyze` step